app/scripts/lib/ppom/ppom-middleware.ts

```diff
@@ -1,6 +1,8 @@
 import { PPOM } from '@blockaid/ppom_release';
 import { PPOMController } from '@metamask/ppom-validator';
 import { NetworkController } from '@metamask/network-controller';
+import { Hex, Json, JsonRpcParams } from '@metamask/utils';
+import { JsonRpcRequest, JsonRpcResponse } from 'json-rpc-engine';
```

Comment:
`JsonRpcRequest` and `JsonRpcResponse` should be imported from `@metamask/utils`. `json-rpc-engine` is actualy re-exporting those two types from `@metamask/utils` and the repo has been archived.

Reply:
ty! updated f47ec1c32e2d02cfa775b6900ce586f32c95be12